docs/UserGuide.md

```diff
@@ -226,27 +264,33 @@ to delete all the sample entries and start using your very own ClientBook.
   <tr>
     <td> Email </td>
     <td> <code>e</code> </td>
-    <td> <ul><li>Should be of the form <code>name@email.com</code></li></ul>
-         <ul><li><code>name</code> part of email should not contain the following characters <code>!#$%&'*+/=?\`{&#x7c;}~^.-</code></li></ul>
-         <ul><li> <code>email</code> part of email should: 
-             <ul><li>Be at least 2 characters long</li></ul>
-             <ul><li>Start and end with either alphabets or numbers</li></ul>
-             <ul><li>Consist only of alphabets and/or numbers, hyphens, or periods between characters</li></ul>
-         </li></ul>
+    <td><ul><li>Should be of the form <code>name@email.com</code></li></ul>
+        <ul><li><code>name</code> part of email should not contain the following characters <code>!#$%&'*+/=?\`{&#x7c;}~^.-</code></li></ul>
+        <ul><li> <code>email</code> part of email should: 
+            <ul><li>Be at least 2 characters long</li></ul>
+            <ul><li>Start and end with either alphabets or numbers</li></ul>
+            <ul><li>Consist only of alphabets and/or numbers, hyphens, or periods between characters</li></ul>
+        </li></ul>
     </td>
   </tr>
   <tr>
     <td> Tag </td>
     <td> <code>t</code> </td>
-    <td> <ul><li>Optional</li></ul> </td>
+    <td><ul><li>Optional</li></ul>
+        <ul><li>Tags can be used to label your relationship with your client e.g. friend, family, classmate</li></ul>
+        <ul><li>Tags associated with a client are shown in alphabetical order in ClientBook</li></ul>
+        <ul><li>If duplicate tags are entered in a command, only 1 will be added to the client.</li></ul>
```

Comment:
Should remove the full stop at the end of this sentence, to be consistent with the other bullet points.